backend/pom.xml

```diff
@@ -37,6 +57,12 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-security</artifactId>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/com.auth0/java-jwt -->
```

Comment:
You guys really need to sort this formatting thing... @MetoMladenovski @DarkoVeljanoskiHtec @DraganMitrasinovicHtec @MarkoRadunovicHtec

Reply:
There is an `Actions on Save` setting in IntelliJ that will automatically do that for you 

![Screenshot 2022-06-06 at 18 18 54](https://user-images.githubusercontent.com/105653778/172202049-d4aec9b1-7c1d-4287-a0ba-a2e5e9675284.png)